atc/atccmd/command.go

```diff
@@ -243,6 +244,10 @@ type RunCommand struct {
 	EnableArchivePipeline bool     `long:"enable-archive-pipeline" description:"Enable /api/v1/teams/{team}/pipelines/{pipeline}/archive endpoint."`
 
 	EnableBuildRerunWhenWorkerDisappears bool `long:"enable-rerun-when-worker-disappears" description:"Enable automatically build rerun when worker disappears"`
+
+	Unsafe struct {
+		MountHolepunches map[string]string `long:"mount-holepunch" description:"Binds a path on all workers into each task step. Can be specified multiple times." value-name:"FROM-PATH:TO-PATH"`
+	} `group:"Unsafe options (optional)"`
```

Comment:
Nice, I think this definitely makes it suitably scary. +1 for holepunch terminology.